libs/ojsonnet/Eval_jsonnet.ml

```diff
@@ -586,53 +580,41 @@ and eval_std_cmp env tk (el : expr) (er : expr) : cmp =
 (* eval_obj_inside *)
 (*****************************************************************************)
 
-and eval_obj_inside env (l, x, r) : V.value_ =
+and eval_obj_inside env (l, x, r) : value_ =
+  let hdupes = Hashtbl.create 16 in
```

Comment:
you should move back hdupes under the match Object since we don't use it in the other case?